functional-programming/FP.md

```diff
@@ -4,10 +4,13 @@
 30-45 minutes
 
 ### Prerequisites
-
+JavaScript basics
```

Comment:
could you add these lessons as markdown links?
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

Reply:
Yes please, they should be linked to the other lessons in this github Techtonica/curriculum repo.